template/.github/workflows/deploy.yml

```
@@ -0,0 +1,33 @@
+name: Netlify Release
```

Should we change this to a name that starts with a verb?
```suggestion
name: Deploy to Netlify
```

---

And should we add a line break the below line as well?

---

50c2b66 ✅